services/admin/backend/infrastructure/repository/project/project_test.go

```diff
@@ -41,48 +41,100 @@ func TestProject_Save(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(5)*time.Second)
 	defer cancel()
 
-	expectedId, err := valueobject.NewIdentifier()
-	expectedShortCode, _ := valueobject.NewShortCode("00FF")
-	expectedShortName, _ := valueobject.NewShortName("short name")
-	expectedLongName, _ := valueobject.NewLongName("project long name")
-	expectedDescription, _ := valueobject.NewDescription("project description")
+	id, err := valueobject.NewIdentifier()
+	shortCode, _ := valueobject.NewShortCode("00FF")
+	shortName, _ := valueobject.NewShortName("short name")
+	longName, _ := valueobject.NewLongName("project long name")
+	description, _ := valueobject.NewDescription("project description")
```

Comment:
I ignore most of the errors here because they've already been tested in their own respective value object unit tests. However, the `Save` function from the repo expects that there might be an error so I need to provide at least one error possibility. I can provide the error value for the rest but they'll be `nil` and checking it is redundant.